### PR TITLE
e2e: verify disabled ics27 host params

### PR DESCRIPTION
**Pull request by gjermundgaraba**
## Description

Extends an existing e2e test to not just check that the host param shows as disabled, but also that the interchain account packets are stopped as expected.

closes: #2118

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against the correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [x] Linked to GitHub issue with discussion and accepted design, OR link to spec that describes this work.
- [x] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [x] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`). (_Comment: N/A_)
- [x] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code). 
- [x] Provide a [conventional commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to follow the repository standards.
- [ ] Include a descriptive changelog entry when appropriate. This may be left to the discretion of the PR reviewers. (e.g. chores should be omitted from changelog) (_Comment: N/A_)
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.
- [ ] Review `SonarCloud Report` in the comment section below once CI passes.

## Summary by CodeRabbit

- **Tests**
	- Enhanced testing for interchain account functionalities, including better validation of token transfers and error handling. Ensured tests cover scenarios before and after disabling the host to verify packet flow control.

### CODE REVIEW

**Review by crodriguezvega — APPROVED** [review]
Looks fabulous to me. Thank you, @gjermundgaraba!

File: e2e/tests/interchain_accounts/params_test.go [hunk 1]
```diff
@@ -124,6 +137,36 @@ func (s *InterchainAccountsParamsTestSuite) TestHostEnabledParam() {
...
+
+		t.Run("start relayer", func(t *testing.T) {
+			s.StartRelayer(relayer)
+		})
+
+		t.Run("verify interchain account", func(t *testing.T) {
+			var err error
+			hostAccount, err = s.QueryInterchainAccount(ctx, chainA, controllerAddress, ibctesting.FirstConnectionID)
+			s.Require().NoError(err)
+			s.Require().NotZero(len(hostAccount))
```
[the remark this answers is not in the document]
Comment (crodriguezvega) [reply e2e/tests/interchain_accounts/params_test.go:158]: Do you actually mean `s.Require().NotEmpty(hostAccount)`?

I think usage of `s.Require().NotZero(len(hostAccount))` is in other tests, so we could change those in one go in a separate, do you agree?

**Review by chatton — APPROVED** [review]
Looks great thank you @gjermundgaraba :1st_place_medal: !

File: e2e/tests/interchain_accounts/params_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (crodriguezvega) [reply e2e/tests/interchain_accounts/params_test.go:158]: https://github.com/cosmos/ibc-go/pull/6076

**Pull request merged**